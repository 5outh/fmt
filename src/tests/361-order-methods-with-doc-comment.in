<?php
//passes:OrderMethod

class A {


	/**
	 * @param $b
	 * @param $c
	 */
	function b($b, $c){

	}


	/* $a */

	/**
	 * @param $a
	 * @param $b
	 */
	function a($a, $b){

	}

}
