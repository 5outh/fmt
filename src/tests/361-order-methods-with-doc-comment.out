<?php
//passes:OrderMethod

class A {

	/**
	 * @param $a
	 * @param $b
	 */
	public function a($a, $b) {

	}

	/* $a */

	/**
	 * @param $b
	 * @param $c
	 */
	public function b($b, $c) {

	}

}
